OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationGenerationJob.java

```diff
@@ -107,10 +108,18 @@ CharSequence getBody() {
 
    /**
     * Get the notification additional data json from the payload
+    * @return
```

Comment:
usually, if the @return is on the java docs you can add a comment in there to add information about the returned object